post-content.php

```diff
@@ -127,10 +129,12 @@
 <!-- /wp:paragraph -->
 
 <!-- wp:gallery {"ids":[null,null],"align":"wide","images":[{"url":"https://cldup.com/_rSwtEeDGD.jpg","alt":""},{"url":"https://cldup.com/L-cC3qX2DN.jpg","alt":""}]} -->
-<ul class="wp-block-gallery alignwide columns-2 is-cropped">
+<figure class="wp-block-gallery alignwide columns-2 is-cropped">
```

Comment:
I had a go at patching up this file locally to see if I could clear the e2e test errors and my diff looked a bit different. The block delimeter comment ended up being updated to:
```
<!-- wp:gallery {"ids":[null,null],"className":"alignwide"} -->
```